README.md

```diff
@@ -63,13 +65,12 @@ The tables lists the core parameters required to generate simulated data using m
 | latent_dim | int: Size of the bottleneck layer|
 | epsilon_std | float: Standard deviation of Normal distribution to sample latent space|
 | validation_frac | float: Fraction of input samples to use to validate for VAE training|
-| num_simulated_samples* | int: If using random sampling approach, simulate a compendia with these many samples|
-| num_simulated_experiments*| int: If using latent-transformation approach, simulate a compendia with these many experiments|
-| num_simulated*| int: If using template-based approach, simulate these many experiments|
-| metadata_delimiter*| str: Delimiter to parse metadata file|
-| metadata_experiment_colname* | str: Column header that contains experiment id that maps expression data and metadata|
-| metadata_sample_colname* | str: Column header that contains sample id that maps expression data and metadata|
-| project_id*| int: If using template-based approach, experiment id to use as template experiment|
+| num_simulated_samples* | int: The number of samples to simulate. This parameter is needed if using the random sampling approach.|
+| num_simulated_experiments*| int: The number of experiments to simulate. This parameter is needed if using the latent transformation approach - simulate_by_latent_transform.|
+| num_simulated*| int: The number of experiments to simulate (i.e. the number of times to shift the starting template experiment). This parameter is needed if using the latent transformation approach - shift_template_experiment.|
+| metadata_experiment_colname* | str: Column header that contains experiment id that maps expression data and metadata. This parameter needed if using either latent transformation approaches.|
+| metadata_sample_colname* | str: Column header that contains sample id that maps expression data and metadata. This parameter needed if using either latent transformation approaches.|
+| project_id*| int: The experiment id to use as template experiment. The id is using the values found in the <metadata_experiment_colname> column of the <metadata_filename>. This <project_id> corresponds to a group of samples that were used to test an single hypothesis. This parameter is needed if using either latent transformation approaches.|
```

Comment:
I think the second sentence here might not make sense? at least the way it is written is hard for me to understand what it means